Zend/zend_hash.c

```diff
@@ -156,6 +160,14 @@ static zend_always_inline void zend_hash_real_init_mixed_ex(HashTable *ht)
 			_mm_storeu_si128((__m128i*)&HT_HASH_EX(data,  8), xmm0);
 			_mm_storeu_si128((__m128i*)&HT_HASH_EX(data, 12), xmm0);
 		} while (0);
+#elif defined(__aarch64__)
```

Comment:
This should use the same check as the header include. Apart from that this LGTM.

Reply:
I will modify the header to be `#if defined(__aarch64__)`.